tests/robot-tests/tests/keywords_and_variables.robot

```diff
@@ -0,0 +1,195 @@
+  run keyword if    ${docker} == 1     Call Method   ${options}   add_argument  --no-sandbox
+
+  create webdriver    Chrome   chrome_options=${options}
+  set window size    1920    1080
+
+user opens chrome without xvfb
+  open browser   about:blank   Chrome
+  maximize browser window
+
+  # bug workaround to maximize window: https://github.com/robotframework/Selenium2Library/issues/677
+  #${options}  evaluate  sys.modules['selenium.webdriver'].ChromeOptions()  sys
+  #call method  ${options}  add_argument  --start-fullscreen
+  #create webdriver  Chrome  chrome_options=${options}
+
+user opens firefox headless
+  ${f_opts} =     Evaluate     sys.modules['selenium.webdriver'].firefox.options.Options()    sys, selenium.webdriver
+  Call Method    ${f_opts}   add_argument    -headless
+  Create Webdriver    Firefox    firefox_options=${f_opts}
+  
+user opens firefox with xvfb
+  start virtual display   1920    1080
+  open browser   about:blank   firefox
+  set window size   1920    1080
+
+user opens firefox without xvfb
+  open browser   about:blank   firefox
+  maximize browser window
+
+user closes the browser
+  close browser
+
+user goes to url
+  [Arguments]   ${destination}
+  go to   ${destination}
+
+user scrolls to the top of the page
+  execute javascript      window.scrollTo(0, 0);
+
+user waits until page contains
+  [Arguments]    ${pageText}
+  wait until page contains   ${pageText}
+
+user waits until page contains element
+  [Arguments]    ${pageText}
+  wait until page contains element  ${pageText}
+
+user waits until page does not contain element
+  [Arguments]    ${pageText}
+  wait until page does not contain element  ${pageText}
+
+user waits until element contains
+  [Arguments]    ${element}    ${text}
+  wait until element contains    ${element}    ${text}
+
+user checks element contains
+  [Arguments]   ${element}    ${text}
+  element should contain    ${element}    ${text}
+
+user checks element does not contain
+  [Arguments]   ${element}    ${text}
+  element should not contain    ${element}    ${text}
+
+user waits until element is visible
+  [Arguments]    ${pageText}
+  wait until element is visible  ${pageText}
+
+user checks element is visible
+  [Arguments]   ${element}
+  element should be visible   ${element}
+
+user checks checkbox is selected
+  [Arguments]    ${checkbox}
+  checkbox should be selected   ${checkbox}
+
+user checks checkbox is not selected
+  [Arguments]    ${checkbox}
+  checkbox should not be selected   ${checkbox}
+
+user waits until element is enabled
+  [Arguments]   ${element}
+  wait until element is enabled   ${element}
+
+user checks element is enabled
+  [Arguments]   ${element}
+  element should be enabled   ${element}
+
+user checks element is disabled
+  [Arguments]   ${element}
+  element should be disabled   ${element}
+
+user clicks link
+	[Arguments]    ${linkToClick}
```

Comment:
Seems like there's inconsistent indentation of arguments from L134 to L167, is this intentional?

Reply:
Not intentional. Fixed